### ISSUE TITLE
Using JEdit as Embedded Editor

### ISSUE THREAD

**Issue by IvanMamontov** [iinegve/muCommander#23]
We can use http://www.jedit.org/api/org/gjt/sp/jedit/textarea/JEditEmbeddedTextArea.html as embedded editor.
Some of jEdit's features include:
- Written in Java, so it runs on Mac OS X, OS/2, Unix, VMS and Windows.
- Built-in macro language; extensible plugin architecture. Hundreds of macros and plugins available.
- Plugins can be downloaded and installed from within jEdit using the "plugin manager" feature.
- Auto indent, and syntax highlighting for more than 200 languages.
- Supports a large number of character encodings including UTF8 and Unicode.
- Folding for selectively hiding regions of text.
- Word wrap.
- Highly configurable and customizable.
- Every other feature, both basic and advanced, you would expect to find in a text editor. See the Features page for a full list.

**Comment by iinegve** [comment]
I like it. Let's give it a chance =)

**Comment by clevertension** [comment]
it seems a bit large requirement

**Comment by iinegve** [comment]
Yeah, that's true, but we didn't have a chance to look at it closely and actually decide if we like it or not. So, if you have spare time, the whole chunk is yours =)
